wordpress-mobile/WordPress-Android
# Move publish to new screen

This PR is the first step of the improved post scheduling. It introduces a new screen that appears when the user clicks on the "Publish" button in post settings. 
In this PR you can click on the "Time and Date" to select a date from Date/Time picker and the label gets filled correctly. Going back to the Post settings sets the scheduled date.
I'll add tests in another PR.

To test:
* Go to Post editor
* Click on Menu/Post settings
* Click on "Publish"
* A new screen opens
* Click on "Time and Date"
* Select a Date from a date picker
* Select a Time from a time picker
* The "Time and Date" label is now set
* Go back
* The "Publish" field has now a label same as the "Time and Date"

Overall notice that the date selection behaviour should be the same as before.

![date_picking_2](https://user-images.githubusercontent.com/1079756/60878891-ef6cc380-a240-11e9-9611-4f36e5bcdf0f.gif)

Update release notes:
